boards/arm/actinius_icarus/board.c

```diff
@@ -40,5 +40,5 @@ static int board_actinius_icarus_init(const struct device *dev)
 	return 0;
 }
 
-SYS_INIT(board_actinius_icarus_init, POST_KERNEL,
-	 CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
+/* Needs to happen after GPIO driver init */
+SYS_INIT(board_actinius_icarus_init, POST_KERNEL, 99);
```

Comment:
Suggest adding a new Kconfig to make this configurable instead of hardcoding the value

Reply:
IMO this operation should not be made configurable as the dependency is that it runs after the GPIO driver init, similar to #33740. Would that be ok?